data/auxiliary/gather/ldap_query/ldap_queries_default.yaml

```diff
@@ -10,6 +10,10 @@ queries:
       - cACertificateDN
       - dNSHostname
       - certificateTemplates
+      - objectGUID
+      - caCertificate
+    referemces:
```

Comment:
```suggestion
    references:
```

Reply:
Fixed in https://github.com/rapid7/metasploit-framework/pull/17071/commits/01e4701bef79a0e7ea19e90431f565324aad3eb2